contracts/governance/Staking/Staking.sol

```diff
@@ -373,6 +477,46 @@ contract Staking is
         emit StakingWithdrawn(msg.sender, amount, until, receiver, isGovernance);
     }
 
+    /**
+     * @notice Send user' staked tokens to a receiver.
+     * This function is dedicated only for governance vesting withdrawal.
+     *
+     * @param amount The number of tokens to withdraw.
+     * @param until The date until which the tokens were staked.
+     * @param receiver The receiver of the tokens. If not specified, send to the msg.sender
+     * @param vesting The vesting address which act as the caller here.
+     * or just unlocked tokens (false).
+     * */
```

Comment:
this note is for `_withdraw` function
it is decided to stick with the direct withdraw only 

- pls remove all governance vesting withdrawal related code from it
- consider extracting common code for both _withdraw and _withdrawDirect into a separate function to keep DRY ;)